test/e2e/e2e_test.go

```diff
@@ -51,6 +51,7 @@ import (
 	_ "k8s.io/kubernetes/test/e2e/lifecycle"
 	_ "k8s.io/kubernetes/test/e2e/lifecycle/bootstrap"
 	_ "k8s.io/kubernetes/test/e2e/network"
+	_ "k8s.io/kubernetes/test/e2e/network/netpol"
```

Comment:
This is an internal sig-network detail.

Please make `test/e2e/network` import `test/e2e/network/netpol`, at which point you can remove all the core `test/e2e` changes and handle this entirely within sig-network

Reply:
this is this way because these tests overlap with other tests under `test/e2e/network` that test exactly the same.
However, until the lack of coverage in the CI for this area , I suggested to start implementing them in a subfolder to do a smooth transition (the test are running in dowmstream CIs), so once we can be sure we don't regress , we just can replace the old ones with this ... 
maybe there is a better approach in go to do this :sweat_smile:

Reply:
ahh, scratch my comment, I see now what you mean, sorry ... this is importing them from the root

Reply:
i think it works this way, just pushed, will confirm shortly

Reply:
If there are test cases in test/e2e/network/netpol IMO it is a good idea to add the `_` import somewhere in test/e2e/network/something.go to ensure that this package always imported

If it is just helper functions then no need.

Reply:
Otherwise some future refactor can remove the netpol.HelperFoo() and then the import netpol and thus also the netpol test cases accidentally

Reply:
ewww circular import error

Reply:
@fejta i tried to add it to 

```
	// import netpol package so that ownership is attributed correctly.
	_ "k8s.io/kubernetes/test/e2e/network/netpol"
```

into the conntrack.go file, as well as the network_policy.go file, both resulted in circular dependency errors...  because this file in `network/netpol` actually imports `"k8s.io/kubernetes/test/e2e/network"` .

Any other suggestions ?

Reply:
oh i have an idea...

```
// SIGDescribe annotates the test with the SIG label.
func SIGDescribe(text string, body func()) bool {
	return ginkgo.Describe("[sig-network] "+text, body)
}
```

can just dupe that 3 liners...

Reply:
done, working now